src/features/Licenses/LicensesFilters/index.jsx

```diff
@@ -0,0 +1,126 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import PropTypes from 'prop-types';
+
+import { Col, Form } from '@edx/paragon';
+import { Select, Button } from 'react-paragon-topaz';
+import { logError } from '@edx/frontend-platform/logging';
+
+import { initialPage } from 'features/constants';
+import { fetchCoursesData } from 'features/Courses/data/thunks';
+import { updateFilters, updateCurrentPage } from 'features/Licenses/data/slice';
+
+import { fetchLicensesData } from '../data';
+
+const LicensesFilters = ({ resetPagination }) => {
+  const dispatch = useDispatch();
+  const institution = useSelector((state) => state.main.selectedInstitution);
+  const courses = useSelector((state) => state.courses.table.data);
+  const licenses = useSelector((state) => state.licenses.table.data);
+
+  const licenseData = useRef(null);
+  const [courseOptions, setCourseOptions] = useState([]);
+  const [licenseOptions, setLicenseOptions] = useState([]);
+  const [courseSelected, setCourseSelected] = useState(null);
+  const [licenseSelected, setLicenseSelected] = useState(null);
+
+  const isButtonDisabled = courseSelected === null && licenseSelected === null;
+
+  const handleSelectFilters = async (e) => {
+    e.preventDefault();
+
+    if (isButtonDisabled) {
+      return;
+    }
+
+    const form = e.target;
+    const formData = new FormData(form);
+    const formJson = Object.fromEntries(formData.entries());
+    const urlParams = Object.entries(formJson)
+      .map(([key, value]) => `${key}=${encodeURIComponent(value)}`)
+      .join('&');
+
+    try {
+      dispatch(updateFilters(formJson));
+      dispatch(updateCurrentPage(initialPage));
+      licenseData.current = licenseOptions;
+      dispatch(fetchLicensesData(institution.id, initialPage, urlParams));
```

Comment:
Why are tou using urlParams as a parameter since it is no necessary in the thunk?
![image](https://github.com/Pearson-Advance/frontend-app-institution-portal/assets/30726391/740ae0d8-bd9d-4dd9-ab12-492b53ed6f9f)

Reply:
In fact, the url params is required, maybe the thunk params isn't clear, so I'll check it

Reply:
url params name is too general and I think this url params are only for filters
```suggestion
      dispatch(fetchLicensesData(institution.id, initialPage, urlParamsFilters));
```